shuup/simple_supplier/forms.py

```diff
@@ -27,3 +27,7 @@ def clean_delta(self):
             raise ValidationError(_("Only non-zero values can be added to stock."), code="stock_delta_zero")
 
         return delta
+
+
+class AlertLimitForm(forms.Form):
+    alert_limit = forms.DecimalField(label=_("Alert limit"))
```

Comment:
This field should include help_text to tell the merchant what this alret limit means. Alert notify event is sent when phycical count goes below alert limit. Or something like that.